packages/corvid-local-site/src/testkit/localSiteBuilder.js

```diff
@@ -205,26 +235,63 @@ const buildFull = (...siteItems) => {
   return fullSite;
 };
 
-const getLocalFilePath = siteItem => {
+const getLocalFilePath = (siteItem, partKey) => {
   const file = itemToFile(siteItem);
+  const pickValue = paths => (isUndefined_(partKey) ? paths : paths[partKey]);
   return sc.matchItem(siteItem, {
-    [sc.pageWithCode]: () => ({ page: file[0].path, code: file[1].path }),
-    [sc.lightboxWithCode]: () => ({
-      lightbox: file[0].path,
-      code: file[1].path
-    }),
+    [sc.page]: () =>
+      pickValue({
+        page: file[0].path,
```

Comment:
@nemni8 	it would be nice not to know about the order of files here. we can do so if the creators will return objects instead of arrays